pkg/multitenant/tenantcostmodel/model.go

```diff
@@ -111,38 +120,46 @@ func (c *Config) ExternalIOIngressCost(bytes int64) RU {
 	return RU(bytes) * c.ExternalIOIngressByte
 }
 
+// KVInterRegionCostMultiplier retrieves the RU multiplier for inter-region
+// transfers between r1 and r2. Ordering does not matter since the model assumes
+// that the multiplier is the same for both directions.
+func (c *Config) KVInterRegionCostMultiplier(r1, r2 string) RUMultiplier {
+	return c.KVInterRegionRUMultiplierTable.CostMultiplier(r1, r2)
+}
+
 // RequestCost returns the cost, in RUs, of the given request. If it is a write,
 // that includes the per-batch, per-request, and per-byte costs, multiplied by
-// the number of write replicas. If it is a read, then the cost is zero, since
-// reads can only be costed by examining the ResponseInfo.
+// the write multiplier. If it is a read, then the cost is zero, since reads can
+// only be costed by examining the ResponseInfo.
 func (c *Config) RequestCost(bri RequestInfo) RU {
 	if !bri.IsWrite() {
 		return 0
 	}
 	cost := c.KVWriteBatch
 	cost += RU(bri.writeCount) * c.KVWriteRequest
 	cost += RU(bri.writeBytes) * c.KVWriteByte
-	return cost * RU(bri.writeReplicas)
+	return cost * RU(bri.writeMultiplier)
```

Comment:
cc: @andy-kimball @jaylim-crl 

There is a design question I think we should discuss. This is applying the writeMultiplier to the request overhead and the network overhead. An alternative approach is we would use a flat count(replicas) multiplier for the request overhead and the variable multiplier for the network overhead.

Most of the cost we are trying to account for is the intra vs inter region networking costs. The CPU cost of a local operation is similar to the CPU cost of a cross-region operation.

I'm in favor of only applying the variable multiplier to the network cost, but I think there is a valid case to be made for applying the cost table to both.

There are some additional costs for the cross region write that would be nice to account for. The cross region write has to be buffered in memory for a longer period of time and is more likely to require retries.